docs/src/components/ReanimatedAnimations/ReanimatedAnimationsSection/styles.module.css

```
@@ -0,0 +1,158 @@
+.animationsSectionWrapper {
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.animationsSectionWrapper > div:first-child {
+  margin-bottom: 7.5rem;
+  background-color: #f5eeff40;
+  width: 100%;
+  height: 23rem;
+  padding: 3rem 7.5rem 3rem 7.5rem;
+}
+[data-theme='dark'] .animationsSectionWrapper > div:first-child {
+  background-color: #473d6840;
+}
+
+.animationsSection {
+  width: 100%;
+  font-size: 1.25rem;
+  color: #fcfcff;
```

```suggestion
  color: var(--swm-white);
```
and use the colors ones already defined in `colors.css`